swift/apple/FirezoneKit/Sources/FirezoneKit/Views/SettingsView.swift

```diff
@@ -578,7 +567,7 @@ public struct SettingsView: View {
         })
       else {
         self.isExportingLogs = false
-        logger.log("Settings window not found. Can't show save panel.")
+        Log.app.log("Settings window not found. Can't show save panel.")
```

Comment:
Well this was to prevent logger initialization from causing a circular dependency issue so I gave up debugging that after a few hours and just fixed it.

Reply:
huh that's interesting. Usually I'd expect global stuff to introduce circular initialization not remove it